ReactNativeClient/lib/services/rest/Api.js

```diff
@@ -451,6 +451,23 @@ class Api {
 			return note;
 		}
 
+		if (request.method === 'PUT') {
+			const note = await Note.load(id);
+			const requestBody = JSON.parse(request.body);
```

Comment:
For consistency we should change the name of 'requestBody' to match what is used in the 'POST' code (just above).

Also there is no need to create this so early in the function, so please move it closer to where it is first used.

Reply:
> Also there is no need to create this so early in the function, so please move it closer to where it is first used.

I have made this change. But, can I know why it is important to not create it early in the function?